.vib/spring-cloud-dataflow/cypress/cypress/fixtures/task-to-import.json

```diff
@@ -0,0 +1 @@
+{"date":1650360393208,"tasks":[{"name":"test-task-imported","dslText":"timestamp","composed":false,"status":"UNKNOWN","composedTaskElement":false,"description":""}]}
```

Comment:
It is not important, but I'd format this to look similar to the other `json` files